RecoMuon/MuonIdentification/plugins/MuonIdProducer.cc

```diff
@@ -637,6 +636,10 @@ void MuonIdProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
      }
    }
 
+   // muon arbitration
+   fillArbitrationInfo( outputMuons.get() );
```

Comment:
It's redundant. Didn't want to break anything removing it completely. Will remove it in future.